src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/dto/OffenderSearch.kt

```diff
@@ -1,13 +1,16 @@
 package uk.gov.justice.digital.hmpps.incentivesapi.dto
 
+import java.time.LocalDate
+
 data class OffenderSearchPrisoner(
   val prisonerNumber: String,
   val bookingId: String,
   val firstName: String,
-  val middleNames: String,
+  val middleNames: String?,
```

Comment:
This fixes the issue that [Andy flagged](https://mojdt.slack.com/archives/C03T6P453CM/p1667392627468919)?

Reply:
yeah, the middle names can be `null` for some reason. We could potentially remove as it's not used at the moment but I didn't as 1) it was there 2) we have first/last name so I thought I'd keep.